examples/MinioClientWithCertificateIdentityProvider.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2021 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import io.minio.MinioClient;
+import io.minio.StatObjectArgs;
+import io.minio.StatObjectResponse;
+import io.minio.credentials.CertificateIdentityProvider;
+import io.minio.credentials.Provider;
+
+public class MinioClientWithCertificateIdentityProvider {
+  public static void main(String[] args) throws Exception {
+    // STS endpoint usually point to MinIO server.
+    String stsEndpoint = "http://STS-HOST:STS-PORT/";
```

Comment:
Since `CertificateIdentityProvider` validates that the `stsEndpoint` must be `https`, it's better to use https in the example too.

Reply:
Fixed